scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/config/SecurityContext.java

```diff
@@ -37,7 +37,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers("/", "/resources/**",
                         "/register").permitAll()
                 // The rest of the our application is protected.
-                .antMatchers("/users/**", "/admin/**").hasRole("ADMIN")
+                .antMatchers("/repositories/**", "/users/**", "/admin/**").hasRole("ADMIN")
```

Comment:
if urls under `/repositories` are only usable by admin I would put them with the prefix `/admin/` and then you don't have to change anything here.